docs/administration-guide/20-custom-maps.md

```diff
@@ -0,0 +1,25 @@
+## Custom Maps
+
+By default, Metabase uses OpenStreetMaps for map visualizations, but there are a few customization options.
+
+### Finding Map Settings
+
+To find the map settings:
+
+1. Click the gear icon in the upper right.
```

Comment:
1. Click the gear icon in the upper right, and then select **Admin**

Reply:
Thanks, fixed!